.env.example

```
@@ -4,3 +4,4 @@ VITE_VAPID_PUBLIC_KEY=your_vapid_public_key
 VAPID_PRIVATE_KEY=your_vapid_private_key
 SUPABASE_AUTH_GOOGLE_SECRET=your_google_auth_secret
 VITE_GA_MEASUREMENT_ID=your_ga_measurement_id
+VITE_SENTRY_DSN=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_SENTRY_DSN key should go before the VITE_SUPABASE_ANON_KEY key

(UnorderedKey)